classes/opponentPhaseBullethell2.lua

```diff
@@ -17,9 +17,9 @@ local BOUNCY_BULLET_SPREAD = 0.3
 local BOUNCY_BULLET_COOLDOWN_TIME = 0.6
 
 
-opponentPhaseBullethell2.RADIUS = 40
+opponentPhaseBullethell2.RADIUS = 35
 opponentPhaseBullethell2.SPAWN_X = ORBIT_FOCAL_X + ORBIT_RADIUS_X * math.cos(ORBIT_START_ANG)
-opponentPhaseBullethell2.SPAWN_Y = ORBIT_FOCAL_Y + ORBIT_RADIUS_Y * math.sin(3 * ORBIT_START_ANG)
+opponentPhaseBullethell2.SPAWN_Y = ORBIT_FOCAL_Y + ORBIT_RADIUS_Y * -0.7 * math.sin(3 * ORBIT_START_ANG)
```

Comment:
Reduced size of phase 2 (maybe should reduce a bit more. 40-45 is probably where we should end up)

Also, multiply height by 0.7 since it was moving way too fast (now that the input ang is multiplied by 3) and made negative so that it spawns on the top half of the screen rather than the bottom half